evopy/evopy.py

```diff
@@ -0,0 +1,48 @@
+"""Module used for the execution of the evolutionary algorithm."""
+import numpy as np
+
+from evopy.individual import Individual
+
+
+#pylint: disable=R0913
+def run(fitness_function, individual_length, warm_start=None, generations=100, population_size=30,
+        num_children=1, mean=0, std=1, maximize=False):
+    """Run the evolutionary strategy algorithm.
+
+    :param fitness_function: the fitness function on which the individuals are evaluated
+    :param individual_length: the length of each individual
+    :param warm_start: the individual to start from
+    :param generations: the number of generations to execute
+    :param population_size: the population size of each generation
+    :param num_children: the number of children generated per parent individual
+    :param mean: the mean
+    :param std: the standard deviation
+    :param maximize: whether the fitness function should be minimized or maximized
+    :return: the best individual found during execution of the algorithm
+    """
+    if individual_length == 0:
+        return None
+
+    if warm_start is None:
+        warm_start = np.zeros(individual_length)
+
+    population = _init_population(population_size, individual_length, mean, std, warm_start)
+    best = sorted(population, reverse=maximize,
+                  key=lambda individual: individual.evaluate(fitness_function))[0]
+    for _ in range(generations):
```

Comment:
Perhaps an empty line before this for-loop? Makes it easier to read

Reply:
Added an empty line before and after for loop